python/core/qgsvectorlayer.sip

```diff
@@ -685,6 +685,11 @@ class QgsVectorLayer : QgsMapLayer, QgsExpressionContextGenerator
     virtual QString getStyleFromDatabase( const QString& styleId, QString &msgError /Out/ );
 
     /**
+     * Will delete the named style corresponding to style id provided from the database
+     */
+    virtual bool deleteStyleFromDatabase( const QString& styleId, QString &msgError );
```

Comment:
Thanks @m-kuhn . I misinterpreted your previous comment.

I've put back the `msgError` available using SIP. I've adjust the python tests according to this signature.